README.md

```diff
@@ -4,9 +4,15 @@
 
 ⚡️ Live at https://ctf.buidlguidl.com
 
-If you have your own stack already, go straight the CTF site and start playing.
+If you have your own stack already, go straight to the CTF site and start playing.
 
-We have created this stack which contains all the tools that you need to play the CTF. Keep reading to learn how to set it up.
+We have created this stack, which contains all the tools you need to play the CTF.
+
+Once you have solved a challenge locally, you can deploy your solution to Optimism, and then interact with the live contracts from the frontend by [switching the network to Optimism.](#nextjs)
```

Comment:
I guess we need to add something about `scripts`?

"If you are using a script, make sure you are pointing it to the Optimism RPC"... or similar?